src/coreclr/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -1575,4 +1590,37 @@ internal static void ValidateObject(object obj, IntPtr pMD)
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern IntPtr NextCallReturnAddress();
     }  // class StubHelpers
+
+#if FEATURE_COMINTEROP
+    internal static class ColorMarshaler
+    {
+        private static readonly MethodInvoker OleColorToDrawingColorMethod;
+        private static readonly MethodInvoker DrawingColorToOleColorMethod;
+
+        internal static readonly IntPtr ColorType;
+
+#pragma warning disable CA1810 // explicit static cctor
+        static ColorMarshaler()
+        {
+            Type colorTranslatorType = Type.GetType("System.Drawing.ColorTranslator, System.Drawing.Primitives", throwOnError: true)!;
+            Type colorType = Type.GetType("System.Drawing.Color, System.Drawing.Primitives", throwOnError: true)!;
```

Comment:
Can this get confused if the current assembly load context is overriden?

The existing code explicitly ignores the ALC overrides: 

https://github.com/dotnet/runtime/blob/d2c19db316b3757c5df60224b55fdc60d6ff69c3/src/coreclr/System.Private.CoreLib/src/System/Reflection/TypeNameResolver.CoreCLR.cs#L154

Reply:
Yes, that could be a problem if the user brings their own custom System.Drawing.Primitives that they override from their ALC (which is an unsupported scenario, but could still be a problem). I'll change this to be more careful.

Reply:
Looking around, we have the same problem in number of places throughout the libraries. I do not think we need to worry about it here. It is just one of many instances.